tests/test_edge_cases.py

```diff
@@ -1113,7 +1118,10 @@ def validator(v):
         (Optional[List[int]], 'Optional[List[int]]'),
         (dict, 'dict'),
         (DisplayGen[bool, str], 'DisplayGen[bool, str]'),
-    ],
+    ]
+    + [(OrderedDict[int, int], 'OrderedDict[int, int]')]
```

Comment:
maybe clearer to just move this to a new test that is skipped unless `OrderedDict` is available.